CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Custom guard error types are now specified as a type of the `StateMachineContext` to allow for
   more versatile types.
 
+### Changes
+
+* [breaking] Actions now take owned values
+* [breaking] `states()` now returns a `Result`
```

Comment:
Actually, it looks like there is already a `Changed` header that should be used. Also, can you merge with `master` to make sure these changes get put under `Unreleased` and not the `v0.6.0` tag?

Reply:
Ready!